doc/api_entity.md

```diff
@@ -216,17 +216,17 @@ A boolean indicating if validation is enabled or not.
 
 ---------
 
-## .getValidation()
+## .getController() (useless)
```

Comment:
The idea was that you could ask the entity if it's valid. Is that still possible?